### PR TITLE
Courseview

### LINKED ISSUE #34: Webview - Course view

### LINKED ISSUE #64: TMC Api: fetch current org information
Fetch Organization data using the slug saved in storage.

### PR DESCRIPTION
**Pull request by samp3**
Resolves #34
Resolves #64

### CODE REVIEW

File: resources/templates/course.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                vscode.postMessage({ type: "setCourse", id });
+            }
+        </script>
+    </head>
+
+    <body>
+        <div class="container container-fluid">
+            <div class="orgHeader row">
+                <div class="col-md-2">
+                    <img class="img-fluid orgImg" src="{{resolve_logo_path organization.logo_path}}" />
```
Comment (jolampi) [inline resources/templates/course.html]: There is some inconsistency with how these classes are named here. Is this a bootsrap/own class separation?

File: src/init.ts [hunk 2]
```diff
@@ -76,7 +98,35 @@ export function registerUiActions(extensionContext: vscode.ExtensionContext, ui:
...
+        if (result.ok) {
+            console.log("Courses loaded");
+            const courses = result.val.sort((course1, course2) => course1.name.localeCompare(course2.name));
+            const organization = resultOrg.val;
+            const data = { courses, organization };
+            ui.webview.setContent(await getTemplate(extensionContext, "course", data));
+        } else {
+            console.log("Fetching courses failed: " + result.val.message);
+        }
+    };
```
Comment (HegePI) [inline src/init.ts]: Exact same functionality in above code.

File: src/api/tmc.ts [hunk 3]
```diff
@@ -80,6 +80,14 @@ export default class TMC {
         return this.checkApiResponse(this.tmcApiRequest("org.json"), createIs<Organization[]>());
     }
 
+    /**
+     * @returns one organization information
+     * @param slug organization slug/id
```
[the remark this answers is not in the document]
Comment (sebazai) [reply src/api/tmc.ts]: Will be fixed today.

File: resources/templates/course.html (hunk 1 above)
Comment (sebazai) [reply resources/templates/course.html]: Will be fixed today.

File: src/init.ts (hunk 2 above)
Comment (sebazai) [reply src/init.ts]: Will be fixed today.

**Pull request merged**